docs/CONTRIBUTING.md

```diff
@@ -61,6 +61,15 @@ are squashed against the correct commit if necessary.
 
 ## Style Guide
 
+### Rust
+
+- Use [`rustfmt`](https://crates.io/crates/rustfmt-nightly) to keep Rust code
+  formatted consistently, according to our [configuration](/.rustfmt.toml).
+- Make sure code is [`cargo clippy`](https://crates.io/crates/clippy) clean.
+  - Check for warnings from
+    [`cargo doc`](https://doc.rust-lang.org/cargo/commands/cargo-doc.html) and
+    [`cargo deadlinks`](https://crates.io/crates/cargo-deadlinks).
+
```

Comment:
That's nice. :) A point that's still a little unclear to me is when to use the individual formatting tools, and when to use our formatting script. — I personally think defaulting to the script for everything might be p straightforward?

Reply:
I agree with @juliettepretot , I think part of the point here is that our "style guide" is first of all whatever the current `format` script does. So I would just add a note along those lines at the top of this doc, and instead use the doc to specify non-enforceable rules (e.g. naming, code location, documentation, etc.).